lms/templates/courseware/progress_graph.js

```diff
@@ -46,40 +57,47 @@ $(function () {
                                               'color' : colors[colorIndex]}
       
       categoryData = categories[ section['category'] ]
-    
+
+      ## Because this is Python (Mako) embedded in JavaScript, our safe linting script is
+      ## thoroughly confused. We should rewrite this file to remove Python/Mako.
+      ## safe-lint: disable=javascript-jquery-append
       categoryData['data'].append( [tickIndex, section['percent']] )
+      ## safe-lint: disable=javascript-jquery-append
       ticks.append( [tickIndex, section['label'] ] )
     
       if section['category'] in detail_tooltips:
+          ## safe-lint: disable=javascript-jquery-append
```

Comment:
Can't fix this-- it's really Python in JavaScript. The linter incorrectly assumes it is a JQuery append.